packages/core/stories/text/text.stories.tsx

```diff
@@ -148,3 +150,52 @@ const LabelCaptionTextComponent: StoryFn<typeof Text> = () => {
 };
 
 export const Label = LabelCaptionTextComponent.bind({});
+
+//********** Notation ***********/
+
+const NotationComponent: StoryFn<typeof Text> = () => {
+  return (
+    <>
+      <NotationText>
+        Notation text - notation - His seasons Shall without form fourth seed
+        so.
+      </NotationText>
+      <br />
+      <NotationText>
+        Notation text
+        <strong> emphasis high</strong>
+      </NotationText>
+      <br />
+      <NotationText>
+        Notation text
+        <small> emphasis low</small>
+      </NotationText>
+    </>
+  );
+};
+
+export const Notation = NotationComponent.bind({});
+
+//********** Action ***********/
+
+const ActionComponent: StoryFn<typeof Text> = () => {
```

Comment:
i'm curious with this, I may have missed the conversation on the decision to make these components themselves (Action and Notation Text)? but wouldn't that mean all Button examples would wrap the text used in <ActionText> ?

Reply:
Convo last week decided we'll have both Notation and Action as Text variant, keeping it in line with how we have variants for all other Text styles as shown in Figma. Not too sure i'm following with the part about Button examples? @libertymayc

Reply:
button uses actions styles, so begs the question whether text within it should use <ActionText> rather than plain text with Button adding the stylign on